config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
+  newName: ghcr.io/aenix-io/etcd-operator
```

Comment:
There is a guide how to develop locally in the repo. I remember that there is a reference to image retagging (simple guide), but from the mobile phone I can't tell exactly what needs to be changed if needed.

Could you please look it through?

Reply:
There's a line in Makefile
```sh
cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
```
it creates an element in `config/manager/kustomization.yaml` `images` list that replcases `controller` image to provided `${IMG}`. If this element already exists, nothing will be changed. If not, new element will be prepended to the list